x-pack/plugins/security_solution/public/common/components/event_details/threat_summary_view.tsx

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiBasicTableColumn } from '@elastic/eui';
+import React, { useMemo } from 'react';
+
+import { TimelineEventsDetailsItem } from '../../../../common/search_strategy';
+import { FormattedFieldValue } from '../../../timelines/components/timeline/body/renderers/formatted_field';
+import { BrowserFields } from '../../../../common/search_strategy/index_fields';
+import { SummaryView } from './summary_view';
+import { getSummaryColumns, Summary, SummaryRow, ThreatSummaryRow } from './helpers';
+import { INDICATOR_DESTINATION_PATH } from '../../../../common/constants';
+
+const getDescription = ({
+  contextId,
+  eventId,
+  fieldName,
+  values,
+}: ThreatSummaryRow['description']): JSX.Element => (
+  <>
+    {values.map((value: string) => (
+      <FormattedFieldValue
+        key={`alert-details-value-formatted-field-value-${contextId}-${eventId}-${fieldName}-${value}-key`}
+        contextId={`alert-details-value-formatted-field-value-${contextId}-${eventId}-${fieldName}-${value}`}
+        eventId={eventId}
+        fieldName={fieldName}
+        value={value}
+      />
+    ))}
+  </>
+);
+
+const getSummary = ({
+  data,
+  timelineId: contextId,
+  eventId,
+}: {
+  data: TimelineEventsDetailsItem[];
+  browserFields?: BrowserFields;
+  timelineId: string;
+  eventId: string;
+}) => {
+  if (!data) return [];
+  return data.reduce<Summary>((acc, { field, originalValue }) => {
+    if (field.startsWith(`${INDICATOR_DESTINATION_PATH}.`) && originalValue) {
+      return [
+        ...acc,
+        {
+          title: field.replace(`${INDICATOR_DESTINATION_PATH}.`, ''),
+          description: {
+            values: Array.isArray(originalValue) ? originalValue : [originalValue],
+            contextId,
+            eventId,
+            fieldName: field,
+          },
+        },
+      ];
+    }
+    return acc;
+  }, []);
+};
+
+const summaryColumns: Array<EuiBasicTableColumn<SummaryRow>> = getSummaryColumns(getDescription);
+
+const ThreatSummaryViewComponent: React.FC<{
+  data: TimelineEventsDetailsItem[];
+  eventId: string;
+  timelineId: string;
+}> = ({ data, eventId, timelineId }) => {
+  const summaryList = useMemo(() => getSummary({ data, eventId, timelineId }), [
+    data,
+    eventId,
+    timelineId,
+  ]);
+
+  return (
+    <SummaryView
+      summaryColumns={summaryColumns}
+      summaryList={summaryList}
```

Comment:
I don't think we need to change this right now, but I wanted to point out the fact that the values in the SummaryView are inline, as opposed to block elements as in the table: <kbd><img width="407" alt="Detections_-_Kibana" src="https://user-images.githubusercontent.com/657252/114118986-5c4a3d00-98af-11eb-8604-478f0203601a.png"></kbd>

I think this works fine, but it's a small difference between "tabular" views, and it doesn't work great with multiple values (which should be a rare case):
<kbd>
<img width="462" alt="Detections_-_Kibana" src="https://user-images.githubusercontent.com/657252/114119490-61f45280-98b0-11eb-8e4d-88c284c0a60d.png">
</kbd>

Reply:
fixed, thanks for noticing

Reply:
Back to block, nice!